server/admin_management/local-dev.env

```diff
@@ -8,5 +8,6 @@ COGNITO_USER_POOL_ID=ca-central-1_p8X8GdjKW
 COGNITO_CLIENT_ID=6jfveou69mgford233or30hmta
 COGNITO_USER_POOL_DOMAIN=dev-fam-user-pool-domain
 FC_API_TOKEN=thisisasecret
-IDIM_PROXY_BASE_URL=https://nr-fam-idim-lookup-proxy-test-backend.apps.silver.devops.gov.bc.ca
+IDIM_PROXY_BASE_URL_TEST=https://nr-fam-idim-lookup-proxy-test-backend.apps.silver.devops.gov.bc.ca
+IDIM_PROXY_BASE_URL_PROD=https://nr-fam-idim-lookup-proxy-prod-backend.apps.silver.devops.gov.bc.ca
```

Comment:
Hi Catherine, maybe this _PROD can be removed?

Reply:
I put it here just because I was testing the prod situation previously on local. I'll remove it.